Cargo.toml

```diff
@@ -25,3 +25,7 @@ features = ['Document', 'DomRect', 'Element', 'HtmlElement', 'Node', 'Window', '
 [dev-dependencies]
 plotters = "^0.3.0"
 wasm-bindgen-test = "^0.3.17"
+
+[dev-dependencies.web-sys]
+version = "0.3.32"
```

Comment:
I would suggest the latest version here, `0.3.58`.

I am not sure the org has a versions policy. In any case, thanks for your contributions.

Reply:
Updated.